src/transport_ofi.c

```diff
@@ -1511,6 +1487,54 @@ int shmem_transport_init(void)
 int shmem_transport_startup(void)
 {
     int ret;
+    int i;
+    int  num_on_node = 0;
+
+    if (shmem_internal_params.OFI_STX_AUTO) {
+
+        ret = shmem_node_util_startup();
+        if (ret != 0) return ret;
```

Comment:
Same comment as earlier, I think I would prefer to startup the node at the top level before transport startup.